### PR TITLE
Force Symfony cache cleans

### PR DESCRIPTION
**Pull request by thewilkybarkid**
It still seems like a PR upgrading Symfony can create problems for builds that upgrade (eg https://alfred.elifesciences.org/blue/organizations/jenkins/pull-requests-projects%2Fjournal/detail/PR-806/5/pipeline) or downgrades (ie separate PR, eg https://alfred.elifesciences.org/blue/organizations/jenkins/pull-requests-projects%2Fjournal/detail/PR-809/1/pipeline)

### CODE REVIEW

File: salt/journal/init.sls [hunk 1]
```diff
@@ -71,6 +71,14 @@ var-directory:
         - require:
             - file: var-directory
 
+journal-cache-clean:
+    cmd.run:
+        - name: rm -rf var/cache/
```
Comment (thewilkybarkid) [inline salt/journal/init.sls]: Or could use `file.directory` and `clean: True`?

**Comment by thewilkybarkid** [comment]
We could also stop relying on Composer scripts for deployment and handle the cache clear etc here.

File: salt/journal/init.sls (hunk 1 above)
Comment (giorgiosironi) [reply salt/journal/init.sls]: The recursion implemented by shell commands is generally faster than the Salt one (which is all Python)

File: salt/journal/init.sls [hunk 2]
```diff
@@ -116,7 +124,7 @@ composer-install:
             - COMPOSER_DISCARD_CHANGES: 'true'
         - require:
             - file: config-file
-            - cmd: var-directory
+            - journal-cache-clean
```
Comment (giorgiosironi) [inline salt/journal/init.sls]: could `SYMFONY_ENV` be removed from this state or mark as deprecated?

Comment (thewilkybarkid) [reply salt/journal/init.sls]: No, the Composer scripts still run. (The cache clean will do nothing and the other script doesn't need an env, but better to safe.)

**Comment by thewilkybarkid** [comment]
Problem is due to the local demo's `parameters.yml` being provisioned too late.

**Comment by thewilkybarkid** [comment]
@giorgiosironi Current build has now hit the critical-CSS problem.

**Comment by giorgiosironi** [comment]
Interrupting the build and disabling that additional required status check, otherwise we can't merge other fixes.

**Comment by giorgiosironi** [comment]
Current required status checks are now `pr-fresh`, `pr-base` and `pr-update`.

**Comment by thewilkybarkid** [comment]
Merge?

**Comment by giorgiosironi** [comment]
Will wait for the current end2end test run checking the new Redis set up in that environment, then merge.

**Pull request merged**